tensorflow/contrib/rnn/python/ops/core_rnn_cell_impl.py

```diff
@@ -83,12 +83,14 @@ def call(self, inputs, state):
 class GRUCell(RNNCell):
   """Gated Recurrent Unit cell (cf. http://arxiv.org/abs/1406.1078)."""
 
-  def __init__(self, num_units, input_size=None, activation=tanh, reuse=None):
+  def __init__(self, num_units, input_size=None, activation=tanh, reuse=None,
+               initializer=None):
     super(GRUCell, self).__init__(_reuse=reuse)
     if input_size is not None:
       logging.warn("%s: The input_size parameter is deprecated.", self)
     self._num_units = num_units
     self._activation = activation
+    self._initializer = initializer
```

Comment:
https://github.com/chris-chris/tensorflow/commit/faa8fef734fab591cd05c1a3e110203e32ada0db

I added kernel_initializer and bias_initializer to GRUCell.
I used kernel_initializer to initialize weights and used bias_initializer to initialize bias.

Could you check this out?